docs/src/documentation/03-components/01-action/textlink/03-accessibility.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/textlink/accessibility/
+---
+
+## Accessibility
+
+The TextLink component has been designed with accessibility in mind.
+
+### Accessibility Props
+
+The following props are available to improve the accessibility of your TextLink component:
+
+| Name          | Type                            | Description                                                                                         |
+| :------------ | :------------------------------ | :-------------------------------------------------------------------------------------------------- |
+| asComponent   | `string \| () => React.Element` | Allows rendering the component as a different element.                                              |
+| ariaCurrent   | `string`                        | Indicates if the component represents the current item within a set of related elements.            |
+| aria-label    | `string`                        | Defines accessible text for the component, replacing the visible text for screen readers.           |
+| aria-controls | `string`                        | Identifies the element controlled by the component, establishing a relationship for screen readers. |
+| title         | `string`                        | HTML attribute providing additional information.                                                    |
+
+The TextLink component extends Common.Globals type, which means it accepts all standard HTML attributes including ARIA attributes. While `ariaCurrent` is implemented as a specific prop, other ARIA attributes (like `aria-controls`, `aria-label`, etc.) can be passed directly as HTML attributes.
```

Comment:
> The TextLink component extends Common.Globals type, which means it accepts all standard HTML attributes including ARIA attributes.

This is not true 🤨 The Common.Globals type only includes `id` and `dataTest` and it is something internal that should not be mentioned in the documentation.

However, it is indeed true that it accepts all data- and aria- attributes, just not because of that inheritance